openmc/mgxs/mgxs.py

```diff
@@ -3891,40 +4201,70 @@ def get_xs(self, in_groups='all', out_groups='all',
             num_mu_bins = 1
 
         # Reshape tally data array with separate axes for domain and energy
-        num_subdomains = int(xs.shape[0] /
-                             (num_mu_bins * num_in_groups * num_out_groups))
-        if self.scatter_format == 'histogram':
-            new_shape = (num_subdomains, num_in_groups, num_out_groups,
-                         num_mu_bins)
+        # Accomodate the polar and azimuthal bins if needed
+        num_subdomains = int(xs.shape[0] / (num_mu_bins * num_in_groups *
```

Comment:
Interestingly there already is a `num_subdomains` property and it doesnt produce the same value as this line on the `get_xs` methods.  Now, that alone doesn't stop me from doing this now, but I think I want to hold off until I formally attack the simplification of Mgxs classes as I discuss in another comment.